unity/EzyClientFactory.cs

```diff
@@ -0,0 +1,38 @@
+using com.tvd12.ezyfoxserver.client.config;
+
+namespace com.tvd12.ezyfoxserver.client.unity
+{
+	public sealed class EzyClientFactory
+	{
+		private static readonly EzyClientFactory INSTANCE = new();
+
+		private EzyClientFactory()
+		{
+		}
+
+		public static EzyClientFactory getInstance()
+		{
+			return INSTANCE;
+		}
+
+		public EzyClient getOrCreateClient(EzyClientConfig config)
+		{
+			var ezyClients = EzyClients.getInstance();
+			var ezyClient = ezyClients.getClient(config.getZoneName());
+			if (ezyClient != null)
+			{
+				return ezyClient;
+			}
+#if UNITY_WEBGL && !UNITY_EDITOR	
+			EzyWSClient ezyWsClient = new EzyWSClient(config);
+			ezyClients.addClient(ezyWsClient);
+			ezyWsClient.init();
+			return ezyWsClient;
+#else
```

Comment:
không phải game nào mình cũng dùng UDP nên có thể nên 1 cờ là UDP_USAGE thì mới toạ UTClient không thì new TcpClient em nhỉ?

Reply:
Cờ UDP_USAGE này nên được truyền vào như nào anh nhỉ?